### ISSUE TITLE
hiproc ends after HiCal and doesn't attempt HiStitch

### ISSUE THREAD

**Issue by AndrewAnnex** [rbeyer/hiproc#5]
* hiproc version: 0.9.0 (current head)
* Python version: 3.8
* Operating System: ubuntu

### Description

I updated Hiproc to v 0.9.0 recently and attempted to run 'PSP_002930_1800' through the pipeline `hiproc -vv -j ./*.IMG`. 

but hiproc just seems to stop without issuing any warning or errors that I can see right after HiCal and does not attempt to run any later streps like HiStitch.

**Comment by rbeyer** [comment]
Hmmm, aha!  The message that may not seem like an error that is, is this: 

```
The image area has a LIS percent (71.098671875) greater than 40, so this image will not be processed through HiCal.
```

Firstly, I need to adjust that message so that it says what image it is, but more importantly, this is a `UserWarning` that gets kicked out by `HiCal` and not handled properly by `hiproc` (it should just cause that one problem image to not be processed, not seize up the whole process).

This should be an easy fix, but may take me a day or two, to get to it.  The interim solution is to run the "pieces" of hiproc [individually](https://hiproc.readthedocs.io/en/latest/processing.html#pipeline-flow), or at least just the initial set: EDR_Stats, HiCal, HiStitch, HiccdStitch, HiColorInit, HiJitReg, and then HiSlither.  The deal here is that the HiCal program itself deals with this error in a way that hiproc (which calls a module from within the HiCal module) doesn't.  Then, from this point, you can run hiproc, and it'll start where HiSlither left off.

**Comment by rbeyer** [comment]
Bleh, no there are some other subtle down-stream problems if we're fundamentally missing a channel, so running things piecemeal may not work like I was hoping.

**Comment by rbeyer** [comment]
This is now fixed in [hiproc 0.10.0](https://github.com/rbeyer/hiproc/releases/tag/v0.10.0) now available on PyPI and soon on conda.
